pushflatpakscript/src/pushflatpakscript/flathub.py

```diff
@@ -16,10 +16,9 @@ def validate_publish_build_output(context, content):
         raise TaskVerificationError("The response from Flathub seems fishy. Bailing out")
 
 
-def run_flat_manager_client_process(context, command, *args):
-    extra_args = [arg for arg in args]
+def run_flat_manager_client_process(context, args):
     flat_manager_client = context.config["flat_manager_client"]
-    process = subprocess.Popen([flat_manager_client, command] + extra_args, stdout=subprocess.PIPE)
+    process = subprocess.Popen([flat_manager_client] + args, stdout=subprocess.PIPE)
```

Comment:
Thanks for fixing this, I initially had a kwargs in my mind and then adjusted but forgot to simply baked it like that 😁

Reply:
:+1: